action.yml

```diff
@@ -33,3 +39,9 @@ runs:
     - id: terraform-docs
       run: echo "version=v0.16.0" >> $GITHUB_OUTPUT
       shell: bash
+    - id: go-mock-version
+      run: echo "version=1.6.0>> $GITHUB_OUTPUT
+      shell: bash
+    - id: go-junit-report-version
+      run: echo "version=2.0.0" >> $GITHUB_OUTPUT
+      shell: bash
```

Comment:
Please put these in alphabetical order with the other steps and remove the `-version` from these `id`s.

Reply:
Please see commit ecbc0939d1f16bce6d60cf088943fd6904f530dc